Automattic/jetpack
# VideoPress: add storage meter to the Admin page

This PR shows the storage meter in the UI:
* Pull the data from the site options
* Stores the value in the state
* Creates a new `<ConnectVideoStorageMeter />` component to print the values

Fixes #

#### Changes proposed in this Pull Request:

* VideoPress: 

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to VideoPress admin dashboard
* Confirm you see the storage meter

<img width="469" alt="image" src="https://user-images.githubusercontent.com/77539/192534231-46ff021c-9d74-469b-baee-9358f8886961.png">


<img width="861" alt="image" src="https://user-images.githubusercontent.com/77539/192534169-6e01b92b-1fd6-4938-bb25-afc2af99b766.png">


Disclaimer: we may need to improve the message when the space used is minimum.

<img width="551" alt="Screen Shot 2022-09-27 at 10 05 51" src="https://user-images.githubusercontent.com/77539/192534781-fadf2520-ced8-4d4c-a70e-78d3ca787469.png">
